app/helpers/green_lanes_helper.rb

```diff
@@ -101,6 +101,16 @@ def exemption_status(exemption, category, category_assessment)
     end
   end
 
+  # Converts a date string to a pretty date string.
+  # Example:
+  #   input: 2024-10-02
+  #   output: 1st Oct 2024
+  def pretty_date(string_date)
+    date = Date.parse(string_date)
+    day_with_suffix = date.day.ordinalize  # Adds 'st', 'nd', 'rd', or 'th' to the day
+    date.strftime("#{day_with_suffix} %b %Y")
```

Comment:
I'm pretty sure the ordinalize stuff is taken care of by our default formatting?

Reply:
i think we can add something to:

# config/initializers/time_formats.rb
`Date::DATE_FORMATS[:pretty_date] = lambda { |date| date.strftime("#{date.day.ordinalize} %b %Y") }`

`date = Date.parse('2024-10-01')`
`date.to_formatted_s(:pretty_date)`

`1st Oct 2024`

Reply:
I've checked with Pranica,
we do not need to use the ordinal number.

Other parts of the site uses just the long version of date: 4 April 2024 (for example)